ihhub/fheroes2
# Remove support for pre-1.0.5 save files

relates to #7132

This will help to maintain much clearer code. We are also playing safe supporting saves up to 6 months old.

How to test this pull request:
- load old save files (from 1.0.5 onwards) and check that everything is normal as in master branch)
- create new save and load them and see the same
